src/backend/app/Http/Controllers/Api/v1/ClientsController.php

```diff
@@ -3,32 +3,91 @@
 namespace App\Http\Controllers\Api\v1;
 
 use App\Http\Controllers\Controller;
-use Illuminate\Http\Request;
+use App\Http\Requests\DestroyClient;
+use App\Http\Requests\EditClient;
+use App\Http\Requests\IndexClient;
+use App\Http\Requests\StoreClient;
+use App\Http\Requests\UpdateClient;
+use App\Models\Client;
+use Illuminate\Http\Response;
+use Illuminate\Support\Collection;
 
 class ClientsController extends Controller
 {
-    //TODO: implement me pls
-    public function index(){}
+    /**
+     * Display data of all clients.
+     *
+     * @param IndexClient $request
+     * @return Collection
+     */
+    public function index(IndexClient $request) : Collection{
```

Comment:
za Collection ti chyba medzera, to potom plati pre vsetky metody, nech to nepisem vsade